npm/BUILD

```diff
@@ -16,19 +16,3 @@
 # specific language governing permissions and limitations
```

Comment:
This package no longer contains any rules since we extracted them into `assemble` and `deploy` subpackages